frontend/src/views/OtherUses/_schema.jsx

```diff
@@ -105,7 +113,101 @@ export const otherUsesColDefs = (optionsData, errors) => [
     suppressKeyboardEvent,
     cellRenderer: (params) =>
       params.value || <Typography variant="body4">Select</Typography>,
-    cellStyle: (params) => StandardCellErrors(params, errors)
+    cellStyle: (params) => StandardCellErrors(params, errors),
+	minWidth: 200
+  },
+  {
+    field: 'provisionOfTheAct',
+    headerComponent: RequiredHeader,
+    headerName: i18n.t(
+      'otherUses:otherUsesColLabels.provisionOfTheAct'
+    ),
+    cellEditor: 'agSelectCellEditor',
+    cellEditorParams: (params) => {
+      const fuelType = optionsData?.fuelTypes?.find(
+        (type) => type.fuelType === params.data.fuelType
+      )
+
+      const provisionsOfTheAct = fuelType
+        ? fuelType.provisionOfTheAct.map((provision) => provision.name)
+        : []
+
+      return {
+        values: provisionsOfTheAct.sort()
+      }
+    },
+    cellRenderer: (params) =>
+      params.value ||
+      (!params.value && <Typography variant="body4">Select</Typography>),
+    cellStyle: (params) => StandardCellErrors(params, errors),
+    suppressKeyboardEvent,
+    minWidth: 300,
+    editable: true,
+    tooltipValueGetter: (p) =>
+      'Select the method for determining carbon intensity'
+  },
+  {
+    field: 'fuelCode',
+    headerName: i18n.t('otherUses:otherUsesColLabels.fuelCode'),
+    cellEditor: AutocompleteCellEditor,
+    cellEditorParams: (params) => {
+        const fuelType = optionsData?.fuelTypes?.find((obj) => params.data.fuelType === obj.fuelType);
+
+        return {
+            options: fuelType?.fuelCodes?.map((item) => item.fuelCode) || [], // Safely access fuelCodes
+            multiple: false,
+            disableCloseOnSelect: false,
+            freeSolo: false,
+            openOnFocus: true
+        };
+    },
+    cellRenderer: (params) => {
+      const fuelType = optionsData?.fuelTypes?.find((obj) => params.data.fuelType === obj.fuelType);
+      if (
+        params.data.provisionOfTheAct === PROVISION_APPROVED_FUEL_CODE &&
```

Comment:
Never use display values as data checks, changing this label will break all your logic.